components/src/components/SimpleEmail/editForm/Component.edit.data.ts

```diff
@@ -1,35 +1,26 @@
+import common from '../../Common/Simple.edit.data';
 export default [
+  ...common,
     {
-        type: 'textfield',
-        label: 'Default Value',
-        key: 'defaultValue',
-        weight: 5,
-        placeholder: 'Default Value',
-        tooltip: 'The will be the value for this field, before user interaction. Having a default value will override the placeholder text.',
-        input: true
-    },
-    {
-        type: 'checkbox',
-        input: true,
-        weight: 70,
-        key: 'delimiter',
-        label: 'Use Thousands Separator',
-        tooltip: 'Separate thousands by local delimiter.'
-    },
-    {
-        type: 'number',
-        input: true,
-        weight: 80,
-        key: 'decimalLimit',
-        label: 'Decimal Places',
-        tooltip: 'The maximum number of decimal places.'
-    },
-    {
-        type: 'checkbox',
-        input: true,
-        weight: 90,
-        key: 'requireDecimal',
-        label: 'Require Decimal',
-        tooltip: 'Always show decimals, even if trailing zeros.'
-    },
+      weight: 200,
+      type: 'radio',
+      label: 'Text Case',
+      key: 'case',
+      tooltip: 'When data is entered, you can change the case of the value.',
+      input: true,
+      values: [
+          {
+              value: 'mixed',
+              label: 'Mixed (Allow upper and lower case)'
+          },
+          {
+              value: 'uppercase',
+              label: 'Uppercase'
+          },
+          {
+              value: 'lowercase',
+              label: 'Lowercase'
+          }
+      ]
```

Comment:
Indentation here looks a bit off. I think this was one of the files that was originally with 4 spaces that you are reformatting to have 2 spaces instead.